### PR TITLE
Feature/13022 ess UI  add further single enc's

### PR DESCRIPTION
**Pull request by pratibh14837**
This PR contains Implementation and Unit test cases for add further single enc's

### CODE REVIEW

File: src/assets/config/appconfig.json [hunk 1]
```diff
@@ -46,7 +46,6 @@
     ]
   },
   "essConfig": {
-    "MaxEncLimit": "250",
-    "MaxEncSelectionLimit":"100"
+    "MaxEncLimit": "250"
```
Comment (crdandoukho) [inline src/assets/config/appconfig.json:49]: I think we need to keep 'MaxEncSelectionLimit' in the config

Comment (pratibh14837) [reply src/assets/config/appconfig.json]: resolved

**Review by crdandoukho — APPROVED** [review]
Looks good

**Review by rvijayavel — APPROVED** [review]
Looks good.

**Pull request merged**